cumulusci/utils/salesforce/tests/test_soql.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import vcr
+
+from cumulusci.utils.salesforce.soql import format_subscriber_package_version_where_clause
+
+
+class TestSoql:
+    spv_id = "04t000000000000"
+
+    @pytest.mark.vcr()
+    def test_format_subscriber_package_version_where_clause_simple(self):
+        where_clause = format_subscriber_package_version_where_clause(self.spv_id)
```

Comment:
This would work in JavaScript, but the Python type-checker wants either an explicit positional argument or a default value in the function definition. (See suggestion above)

Reply:
Thanks! I fixed this locally after figuring out how to launch `pytest`, but hadn't pushed it yet. I'm definitely a little fresh to Python so I appreciate the guidance!